internal/commands/result.go

```diff
@@ -399,16 +399,17 @@ func parseResultsSonar(results *wrappers.ScanResultsCollection) []wrappers.Sonar
 		for _, result := range results.Results {
 			auxIssue.Severity = severities[result.Severity]
 			auxIssue.Type = vulnerabilitySonar
-			// CxSAST result type
-			if result.Type != kicsTypeLabel {
+
+			if strings.EqualFold(strings.TrimSpace(result.Type), sastTypeLabel) {
 				auxIssue.EngineID = result.Type
```

Comment:
Could we do this outside the if to avoid dup with the next if or is there conflict with SCA?

Reply:
It shouldn't have impact on SCA. I will put it above